src/options.html

```diff
@@ -1,60 +1,61 @@
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">   
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
```

Comment:
and what about these lines? Why are you editing them without any changes?

Reply:
@zamhaq I think he just removed extra space thats why its shown in edited.